main.js

```diff
@@ -313,6 +311,62 @@
       route: "/typography",
       component: <UXWTypography />,
     },
+    {
+      heading: "Component",
+      subheadings: [
+        {
```

Comment:
You should be able to delete this empty entry. You don't have a subtitle or anchor anyway.

Reply:
updated